less/documentDB.less

```diff
@@ -2576,6 +2576,10 @@ a:link {
 .querydropdown.placeholderVisible {
   font-style: italic;
 }
+.querydropdown.placeholderVisible::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
+  color: #000;
```

Comment:
I don't think the placeholder text should be in black. It will be the same color as the actual text.
![image](https://user-images.githubusercontent.com/56978073/226708618-cefe8e1d-a719-4589-8081-a238be326368.png)
![image](https://user-images.githubusercontent.com/56978073/226708669-4bd4736e-963e-44d4-b87d-7eee5bf612e1.png)

If the original color was too light, you can use a darker gray.

Reply:
I will use color with less contrast compared to text color used and make relevant changes for placeholder color and make a commit.